src/main/scala/Executor.scala

```diff
@@ -24,5 +24,5 @@ trait Executor {
 }
 
 case class Result(
-  finalValue: AnyVal
+  finalValue: Option[List[AnyVal]]
```

Comment:
Haha, but *you* were the one who made it AnyVal ;) but yes, having the type would be good how are you suggesting we specify the type here, I don't quite understand